protected/humhub/modules/comment/resources/js/humhub.comment.js

```diff
@@ -251,36 +251,52 @@ humhub.module('comment', function (module, require, $) {
         });
     };
 
-    var toggleComment = function(evt) {
+    var toggleComment = function (evt) {
         var visible = evt.$target.is(':visible');
+        var userName = '@' + evt.$target.closest('.media').find('.media-heading a').html() + ' ';
+        var $form;
+        var target;
+
+        function focusOnForm() {
+            target.find('.humhub-ui-richtext:last').trigger('focus');
+            target.find('.humhub-ui-richtext').find('p').text(userName);
+            target.find('.humhub-ui-richtext:last').trigger('focus');
+        }
 
         // Comments are shown but form is not visible yet --> Toggle form only
         if (visible && !evt.$target.children('.comment_create').is(':visible')) {
             evt.$target.children('.comment_create').slideToggle(undefined, function() {
-                evt.$target.find('.humhub-ui-richtext').trigger('focus');
-
+                focusOnForm(evt.$target, userName);
             });
             return;
         }
 
-        var $form =  evt.$target.children('.comment_create');
+        // Check witch 'Replay' button pressed under first or second level of comments
+        if (evt.$target.parents('.nested-comments-root').length !== 2) {
```

Comment:
Maybe it would be better to implement a custom action beside toggleComment in case of sub comment reply? Something like `toggleParentComment` which just calls toggleComment of the parent.